spec/classes/init_spec.rb

```diff
@@ -1,169 +1,174 @@
 require 'spec_helper'
```

Comment:
Please unstage this file. I'm going to redo these spec tests to make them easier to understand and contribute to then will add tests for this PR after rebasing.